src/jvm/test/resources/expected_logs/suspend_trace_reporting.txt

```diff
@@ -22,10 +22,10 @@ The following interleaving leads to the error:
 |   barStarted ➜ true at SuspendTraceReportingTest.foo(SuspendTraceReportingTest.kt:29)                                                                       |                                                                                                                                          |
 |   canEnterForbiddenBlock = true at SuspendTraceReportingTest.foo(SuspendTraceReportingTest.kt:29)                                                           |                                                                                                                                          |
 |   $i$a$-withLock$default-SuspendTraceReportingTest$foo$2.lock(null): COROUTINE_SUSPENDED at SuspendTraceReportingTest.foo(SuspendTraceReportingTest.kt:184) |                                                                                                                                          |
```

Comment:
Please raise an issue for this weird function call

Reply:
#625